requirements/fabric/test.txt

```diff
@@ -7,3 +7,4 @@ pytest-random-order ==1.1.0
 click ==8.1.7
 tensorboardX >=2.2, <2.7.0  # min version is set by torch.onnx missing attribute
 torchmetrics >=0.7.0, <1.3.0 # needed for using fixed compare_version
+bitsandbytes ==0.41.0  # strict
```

Comment:
Would it make sense to have it in https://github.com/Lightning-AI/pytorch-lightning/blob/master/requirements/fabric/strategies.txt ?

Reply:
Why? what's the difference?